draft/2025-04-02-this-week-in-rust.md

```diff
@@ -44,7 +44,8 @@ and just ask the editors to select the category.
 ### Research
 
 ### Miscellaneous
-
+* [Public mdBooks](https://mdbooks.code-maven.com/)
```

Comment:
```suggestion
* [Public Rust mdBooks](https://mdbooks.code-maven.com/)
```

Reply:
This addition might make it a little more clear that these are all Rust-related

Reply:
Except that not all of them are Rust related. Most of them are, but the point of the site is to collect all the (public) users of `mdbooks` which is a Rust project.

Reply:
Hi @szabgab thanks for the clarification. It sounds like this is collecting users of the mdBooks Rust project, so it makes sense as a project update story.

Reply:
This can stay in Misc as-is, or be moved to Project updates, up to you.